server/src/main/java/com/objectcomputing/checkins/security/GoogleServiceConfiguration.java

```diff
@@ -1,171 +1,64 @@
 package com.objectcomputing.checkins.security;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import io.micronaut.context.annotation.ConfigurationProperties;
-import io.micronaut.core.annotation.Introspected;
 import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
 
+@Getter
+@Setter
 @ConfigurationProperties("service-account-credentials")
-@Introspected
 public class GoogleServiceConfiguration {
 
     @NotNull
-    public String directory_id;
+    @JsonProperty("directory_id")
```

Comment:
Not sure your plan for this PR after our discussion but, if you don't end up removing these completely, it's worth noting that you converted the `_`s to `-`s in the json.

Reply:
In the *yaml* configuration I converted to use hyphens instead of underscores, but both will work for defining properties in Micronaut (but hyphen is more idiomatic)

In the generated JSON from this object, it will still use underscores (as the `JsonProperty` specifies underscores)